CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Cassava Megaparsec 2.1.0
+
+* Drop support for GHC<8.2.2
```

Comment:
Strictly, this would mean you also drop GHC 8.2.1, but this package would build there.
Maybe: "Drop support for GHC 8.0 and below".

Reply:
You are right your suggested message is more precise.